### PR TITLE
DEV: LEAF-2216 - Fixed Sync Services Re-Add

### PR DESCRIPTION
**Pull request by pete-nerantzinis**

### CODE REVIEW

**Review by Vaceek — APPROVED** [review]

File: LEAF_Request_Portal/admin/templates/mod_svcChief.tpl [hunk 1]
```diff
@@ -158,7 +158,7 @@ function removeUser(groupID, userID) {
     });
 }
 
-function initiateWidget(serviceID) {
+function initiateWidget(serviceID, serviceName) {
```
Comment (gsteinhoff) [inline LEAF_Request_Portal/admin/templates/mod_svcChief.tpl:161]: Are we sure that this function call will always have a value in all calls to this function now?  If not, we need to overload to a set value that would be its default.  Not failing, just asking if considered.

Comment (pete-nerantzinis) [reply LEAF_Request_Portal/admin/templates/mod_svcChief.tpl:161]: Yeah this function is only created and used in this file. But I probably should add a default value of blank? and 0?

Comment (gsteinhoff) [reply LEAF_Request_Portal/admin/templates/mod_svcChief.tpl:161]: I would add a default that you know should never be like null, false, etc to match the data type it should be.  So if string then maybe empty string or null would work the best.  Then, you can check for your default and reject the request outright if required and that value.  if optional value, you can just check it has good data before it used (as you usually would).

Comment (pete-nerantzinis) [reply LEAF_Request_Portal/admin/templates/mod_svcChief.tpl:161]: Yeah I agree and it is checking there is data before passing it. As per the ajax request being successful it shows it has data and then passes it on line 267.

**Review by gsteinhoff — APPROVED** [review]
Everything seems OK to me.

**Review by Pelentan — APPROVED** [review]
Looks good.

**Pull request merged**